shared-components/tl-tournament-card/tl-tournament-card-css.js

```diff
@@ -0,0 +1,79 @@
+import { css } from 'lit';
+
+export const tournamentCardStyles = css`
+    .details-button {
+        justify-content: center;
+        align-items: center;
+        width: 10em;
+        min-height: 4em;
+    }
+    .image-date-wrapper {
+        position: relative;
+    }
+    #date-time-card {
+        position: absolute;
+        top: 0;
+        right: 0;
+        margin: 0 3.6em;
+        z-index: 1;
+    }
+    .tournament-img {
+        position: absolute;
+        top: 0;
+        margin-top: 1em;
+        width: 100%;
+    }
+    .prize-wrapper {
+        display: grid; 
```

Comment:
Why a `display: grid` here where the organization is just horizontal? Why not a `display: flex`? 

IMHO, the parent `.tournament-wrapper` which is set for `flex` could have benefit from `grid`... Not this one.

Reply:
I totally agree.